pkg/util/dynamichelper/dynamichelper_test.go

```diff
@@ -8,15 +8,26 @@ import (
 	"testing"
 	"time"
 
+	mcv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
+	extensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	extensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	kruntime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/intstr"
 
+	arov1alpha1 "github.com/Azure/ARO-RP/pkg/operator/apis/aro.openshift.io/v1alpha1"
 	"github.com/Azure/ARO-RP/pkg/util/cmp"
 )
 
 func TestMerge(t *testing.T) {
 	serviceInternalTrafficPolicy := corev1.ServiceInternalTrafficPolicyCluster
+	var terminateGracePeriod int64 = corev1.DefaultTerminationGracePeriodSeconds
+	var revisionHistoryLimit int32 = 10
+	var replicasDeployment int32 = 1
+	var progressDlineSec int32 = 600
```

Comment:
Overall this lgtm but I have one small nit. We should rename this variable to `progressDeadlineSec` or something similar.